src/node-settings/node-attributes/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Basic Settings'
+description: 'Basic Settings'
+keywords: 'abtnode, blocklet'
+author: 'wangshijun'
+category: ''
+layout: 'documentation'
+tags:
+  - 'abtnode'
+---
+
+The *Basic Settings* view configured the below described attributes about the ABT Node.
+
+ ![](./images/node-settings-2-en.png)
+
+Besides the ABT Node details, the settings page also provides the end-user license agreement.
+
+### Name Your Node
+
+The node name is displayed at the top of the node console page to identify which node they are accessing.
+
+### Describe Your Node
+
+It is displayed at the top of the node console page.
+
+### Register Your Node
+
+Allows users to reach your ABT Node.
+
+### DID Wallet Location
+
+Provides the location of DID wallet.
```

Comment:
We should make it clear that this url points to the url of an deployed DID-Wallet blocklet, maybe we should add link to that blocklet.